### DISCUSSION TITLE
Does Lapce support Chinese? How to set the font?And how to make Figure 1 look like Figure 2?

### DISCUSSION THREAD

**Discussion by Draven-1** [lapce/lapce#3356] [Q&A]
1. 
![1](https://github.com/lapce/lapce/assets/54003179/00a95787-bbdf-4b97-96dc-5d97a52e075b)

2. 
![2](https://github.com/lapce/lapce/assets/54003179/27e2d519-c16f-4cc0-8f4f-2e1f67b313c4)

**Comment by Bevisy** [comment]
try Terminal: Font Family setting
![image](https://github.com/lapce/lapce/assets/8726659/72b0db30-14d4-4472-b82c-30a4ebe17c05)

**Comment by Neutron3529** [comment]
Use `fc-list  | grep -i mono` could give you informations about the correct font name to use.

For me, install `wqy font family` by `pacman -S wqy-microhei` and then use `WenQuanYi Micro Hei Mono` should be fine.
